hack/build/run-functional-tests.sh

```diff
@@ -29,6 +29,8 @@ KUBECONFIG=${KUBECONFIG:-${CDI_DIR}/cluster/.kubeconfig}
 GOCLI=${GOCLI:-${CDI_DIR}/cluster/cli.sh}
 KUBE_MASTER_URL=${KUBE_MASTER_URL:-""}
 CDI_NAMESPACE=${CDI_NAMESPACE:-cdi}
+SNAPSHOT_SC=${SNAPSHOT_SC:-csi-rbd}
```

Comment:
I'm wondering if there's a better way to tackle this than specifying each SC by feature.  For now it's not horrible to have two; but I see this growing and getting unwieldy over time.  We could introduce *special* requirements for functional testing storage classes whereby we include annotations in the sc itself on what features it supports.

It's not great either, maybe there's a better solution; but I think dumping in more env variables to call the test with may be ugly after a while.

I do agree with basically making all of these default to the csi-rbd sc since we're deploying that in the dev environment.

Reply:
The problem with annotating storage classes is how do we handle the case where multiple storage classes carry the same annotations?

Reply:
So I think it should be manageable, since certain features will be mainlined and every storage class will have them at some point, then we can remove that feature parameter. So I am expecting a maybe up to half a dozen or so features, that will exist in alpha/beta for a while at a time.

I think @j-griffith is fine with this at this point since we discussed it offline a little more.